openfe/setup/atommapping.py

```diff
@@ -1,10 +1,15 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/openfe
 from dataclasses import dataclass
-from typing import Dict, TypeVar
+from itertools import chain
+from typing import Dict, NewType
+
+from rdkit import Chem
 
 from openfe.setup import Molecule
 
+AtomMap = NewType("AtomMap", Dict[int, int])
```

Comment:
I'm not sure what this alias adds?

Reply:
It saves from having to use `Dict[int, int]` whenever we have an atom map type, I was thinking we could have a `openfe/utils/types.py` to keep track of our types, I am new to using typing so if this isn't a good idea/not best practices then we shouldn't do it.

Reply:
Sounds good! I'm pretty new to using type hints so I thought it would be good to save having to type `Dict[int, int]` each time but I see your point! I'll remove it.